### DISCUSSION TITLE
Flutter widget testing with Nylo

### DISCUSSION THREAD

**Discussion by 1stBoltCrux** [nylo-core/nylo#56] [Q&A] [answered]
Hey! Love the framework but had a question. Are there any resources available for setting up widget testing in a repo that's utilizing Nylo?

Pretty new to flutter testing and I'm having a tough time mocking or providing methods to my test widgets that rely on the framework.

Thanks in advance for the assistance!

**Comment by agordn52** [comment]
@1stBoltCrux thanks for the comment, I'll get back to you on this at my earliest convenience.

**Reply by agordn52** [reply, marked as answer]
Hi @1stBoltCrux,

At the moment there are no docs on https://nylo.dev for testing, however, you can check out this guide [here](https://docs.flutter.dev/cookbook/testing/widget/introduction) to learn how to test widgets.
